### PR TITLE
Remove extra element from DOM when prop is invisible

### PR DESCRIPTION
**Pull request by Sinan997**
Resolves https://github.com/abpframework/abp/issues/19167

### CODE REVIEW

File: npm/ng-packs/packages/components/extensible/src/lib/components/extensible-form/extensible-form.component.ts [hunk 1]
```diff
@@ -44,11 +44,13 @@ export class ExtensibleFormComponent<R = any> {
         const type = !record || JSON.stringify(record) === '{}' ? 'create' : 'edit';
         const propList = this.extensions[`${type}FormProps`].get(this.identifier).props;
         this.groupedPropList = this.createGroupedList(propList);
+        this.groupedPropListOfArray = this.groupedPropList.items.map(i => i.formPropList.toArray());
```
Comment (masum-ulu) [inline npm/ng-packs/packages/components/extensible/src/lib/components/extensible-form/extensible-form.component.ts]: Do we really need this ? we can already reach `formPropList` property in `groupedPropList` use this in `isAnyGroupMemberVisible` ?

Comment (Sinan997) [reply npm/ng-packs/packages/components/extensible/src/lib/components/extensible-form/extensible-form.component.ts]: `formPropList` is linked list, so i need to convert this to array so i can traverse it easly

Comment (masum-ulu) [reply npm/ng-packs/packages/components/extensible/src/lib/components/extensible-form/extensible-form.component.ts]: My problem not about casting, it's about the why you define new variable for this ? You using only in `isAnyGroupMemberVisible`

Comment (Sinan997) [reply npm/ng-packs/packages/components/extensible/src/lib/components/extensible-form/extensible-form.component.ts]: i understood what you mean, i will remove unnecesarry variable

**Review by masum-ulu — APPROVED** [review]

**Pull request merged**